Jenkinsfile

```diff
@@ -19,9 +19,22 @@ pipeline {
     }
 
     stage('Run tests') {
-      steps {
-        sh './ci/test.sh'
-        junit 'tests/junit/*'
+      parallel {
+        stage("Test conjur_lookup Plugin") {
+          agent { label 'executor-v2-large' }
```

Comment:
It doesn't, but I ran into issues with all the test containers running on the same executor and this was the only solution that I had at the time. I'm not sure how to fix it otherwise right now.

Reply:
was it just midair collisions due to not setting `COMPOSE_PROJECT_NAME` or something else?

Reply:
That might be it, I'll investigate

Reply:
This has been resolved! The COMPOSE_PROJECT_NAME was being set in the individual test scripts for the plugin and role, which overrode the project name I was assigning. I changed them a bit so that they won't confuse docker.